mart/attack/callbacks/base.py

```diff
@@ -8,9 +8,7 @@
 from __future__ import annotations
 
 import abc
-from typing import TYPE_CHECKING, Any, Dict, Union
-
-import torch
+from typing import TYPE_CHECKING
 
```

Comment:
I would revert all these changes and make the callbacks keyword only parameters.